app/javascript/onboarding/components/__tests__/FollowTags.test.jsx

```diff
@@ -162,4 +162,59 @@ describe('FollowTags', () => {
       expect(getByText('0 tags selected')).toBeInTheDocument(),
     );
   });
+
```

Comment:
Shouldn't we also test that the corresponding `fetch` should happen?

Reply:
Thanks @lightalloy I have added one last test to cover the API call too.